pipelines/nextflow/subworkflows/additional_seq_prepare.nf

```diff
@@ -43,4 +45,11 @@ workflow additional_seq_prepare {
         manifested_dir = MANIFEST(collect_dir, accession)
         manifest_checked = CHECK_INTEGRITY(manifested_dir, brc_mode)
         PUBLISH_DIR(manifest_checked, accession)
+        manifeststats_ch = manifeststats(manifested_dir, accession, 'datasets')
+}
+
+workflow {
+    additional_seq_prepare(params.prefix, params.accession, params.production_name, params.brc_mode)
+    additional_seq_prepare.out//.manifeststats_ch
+        | view()
```

Comment:
This might be simpler to read/handle:
```suggestion
    manifest_stats = additional_seq_prepare(params.prefix, params.accession, params.production_name, params.brc_mode)
    manifest_stats.view()
```